metadetect/fitting.py

```diff
@@ -396,7 +396,7 @@ def _sum_bands_wavg(
 
                 raw_mom += (_wgt * res[MOMNAME] / mom_norm)
                 raw_mom_cov += (_wgt**2 * res[MOMNAME+"_cov"] / mom_norm / mom_norm)
```

Comment:
I think it would be more clear to leave `wgt` in there and explicitly put flux ratio in, with a comment explanation that this is unity for the object.  In other words do away with `_wgt` which visually is hardly distinguishable from `wgt`